src/conversions.jl

```diff
@@ -47,20 +47,20 @@ convert(::Type{RGB24},   x::Real) = RGB24(x)
 convert(::Type{ARGB32},  x::Real) = ARGB32(x)
 convert(::Type{Gray24},  x::Real) = Gray24(x)
 convert(::Type{AGray32}, x::Real) = AGray32(x)
-convert(::Type{ARGB32},  x::Real, alpha) = ARGB32(x, alpha)
```

Comment:
For some reason I have the sneaking suspicion that removing this method is causing the 0.5 error